datasets/steam/setup_steam_instance/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "setup_steam_instance",
+  "version": "1.0.0",
+  "description": "",
+  "main": "script.js",
+  "scripts": {
+    "setup": "node script.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "meilisearch": "^0.24.0"
```

Comment:
This will need to be kept up to date with the latests versions of meilisearch-js